src/javascript/binary/chart.js

```diff
@@ -0,0 +1,4 @@
+var chart;
```

Comment:
@raunakkathuria there was a file called livechart and I removed everything else in it and changed the name to chart, but I think we need these vars as they are used elsewhere. should I move them to another file?